src/rocm_docs/projects.py

```diff
@@ -277,18 +278,68 @@ def _get_context(
     }
 
 
+def _update_banner_config(
+    app: Sphinx, current_project: str, branch: str, url: str
+) -> None:
+    schema_file_loc = "data/projects.schema.json"
+    schema_file = importlib_resources.files("rocm_docs") / schema_file_loc
+    with open(schema_file) as file:
+        schema_config = yaml.safe_load(file)
+
+    mapping_file_loc = "data/projects.yaml"
+    mapping_config = importlib_resources.files("rocm_docs") / mapping_file_loc
+    with open(mapping_config) as file:
+        project_dict = yaml.safe_load(file)
+
+    development_branch = schema_config["$defs"]["project"]["properties"][
+        "development_branch"
+    ]["default"]
+    print(project_dict["projects"][current_project])
+
+    if (
+        type(project_dict["projects"][current_project]) is dict
+        and "development_branch"
+        in project_dict["projects"][current_project].keys()
+    ):
+        development_branch = project_dict["projects"][current_project][
+            development_branch
+        ]
+
+    latest_version = "5.5.1"
+    latest_version_string = f"docs-{latest_version}"
+    latest_url = re.sub("([^\/]+$)", latest_version_string, url)
+    announcement_info = ""
+
+    if branch == latest_version_string:
+        announcement_info = "This is the latest version of ROCm documentation."
+    elif branch.startswith("docs-"):
+        announcement_info = (
+            f"This is an old version of ROCm documentation. The latest version is <a href='{latest_url}'>{latest_version}</a>",
```

Comment:
Documentation for the latest Linux release of ROCm is available here.

Reply:
This is documentation for an older release of ROCm. Read the latest ROCm release documentation to stay informed of all our developments.

Reply:
[533592c](https://github.com/RadeonOpenCompute/rocm-docs-core/pull/261/commits/533592c39dff68d794c83632fe19d03da01d38aa)